frontend/simple/js/state.js

```diff
@@ -5,22 +5,146 @@
 
 import Vue from 'vue'
 import Vuex from 'vuex'
-
-const S = require('string')
+import EventLog from './event-log'
+import {EVENT_TYPE} from '../../../shared/constants'
+import {makeUserSession} from '../../../shared/functions'
+import localforage from 'localforage'
+import _ from 'lodash-es'
+const {SUCCESS, ERROR} = EVENT_TYPE
 
 Vue.use(Vuex)
-
+let _userSessions = localforage.createInstance({
```

Comment:
Can this be renamed to `appSessionData`, and maybe a comment to explain that it stores data across launches of the app?